package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@gridsuite/commons-ui",
-    "version": "0.64.1",
+    "version": "0.64.2",
```

Comment:
To remove the release new version in the same PR for commons-ui, should do in a separated PR pour the reason of traceability